### ISSUE TITLE
LA Consumption: ADF Managed Identity connection loses its connectionProperties

### ISSUE THREAD

**Issue by xuehongg** [Azure/LogicAppsUX#2210]
### Describe the Bug with repro steps

1. Create a workflow and enable managed identity
2. Add an ADF action and create new connection for the ADF action with user sign-in, and it saves successfully
3. Open the ADF action to change to a new connection. This time create a connection with managed identity, and it saves successfully.
4. Switch back to the user sign-in connection created in step 2, and it still saves successfully.
5. Switch back to the managed identity connection created in step 3, now save fails with this error.

![image](https://user-images.githubusercontent.com/49498706/235811685-4e4c8f52-dc62-44ad-adee-be33812943ce.png)

Basically during the above repro process, the circled connection properties is lost.

![image](https://user-images.githubusercontent.com/49498706/235811978-d9d299a8-5966-4c06-bbfe-2814240007a0.png)

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using Preview Designer or GA Designer

Generally Available

### Browser

Chrome

**Comment by Eric-B-Wu** [comment]
@preetriti1 Tony had mentioned that this was something you had previously looked at, and it needed to first be supported by the backend.

**Comment by preetriti1** [comment]
I have no idea what you are talking about... can you give me little context on this? Ideally managedServiceidentity connection is supported in backend so what did Tony mention?

**Comment by rllyy97** [comment]
Saving consumption workflows with Managed Identity Connections was bugged in preview up until a recent change I made in Portal.  I think the original issue was marked as GA incorrectly as we haven't made any changes to GA in a while and I'm unable to repro.
I just tested the repro steps in preview with my latest changes and this issue is not longer occurring.
